docs/networking.md

```diff
@@ -62,17 +68,11 @@ CNI networking providers are not part of the Kubernetes project, we do not maint
 their installation processes.  With that in mind, we do not support problems with
 different CNI providers but support configuring Kubernetes to run CNI providers.
 
-## Specifying network option for cluster creation
-
-You are able to specify your networking type via command line switch or in your yaml file.
-The `--networking` option accepts the three different values defined above: `kubenet`, `cni`,
-`classic`, and `external`. If `--networking` is left undefined `kubenet` is installed.
-
 ### Weave Example for CNI
 
```

Comment:
So this is a example how to use --networking cni - maybe we need to rewrite this ;) Please blackout your changes.